sdk/compute/arm-compute-profile-2020-09-01-hybrid/samples-dev/availabilitySetsCreateOrUpdateSample.ts

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
hmm, maybe we should fix these multiple license headers in a later PR.

Reply:
Yes, I'll do a cleanup on that next.